pandamium_datapack/data/pandamium/functions/startup.mcfunction

```diff
@@ -61,13 +61,15 @@ scoreboard objectives add staff_world trigger
 scoreboard objectives add tp_pre_jail trigger
 scoreboard objectives add player_info trigger
 scoreboard objectives add container trigger
+scoreboard objectives add hide trigger
 
 scoreboard objectives add spawnpoint_x dummy
 scoreboard objectives add spawnpoint_y dummy
 scoreboard objectives add spawnpoint_z dummy
 scoreboard objectives add spawnpoint_dim dummy
 scoreboard objectives add active_particles dummy
 scoreboard objectives add selected_player dummy
+scoreboard objectives add hidden dummy
```

Comment:
not sure we want to reset this, since it means the player will unhide after an autorestart

Reply:
Currently only the trigger is reset, so should be fine

Reply:
Oh wait I'm stupid, I didn't read properly and mixed it with the other line